.circleci/config.yml

```diff
@@ -22,6 +22,15 @@ jobs:
           name: "run cargo tests"
           command: cargo test --locked
 
+  bench-test:
+    docker:
+      - image: quay.io/influxdb/wasm-build
+    steps:
+      - checkout
+      - run:
+          name: "run cargo bench"
+          command: cargo bench
```

Comment:
`cargo bench` will compile in release mode which can be pretty slow, if we just want to test that they work then `cargo test --benches` will run the benchmarks in test mode instead (or add `--all-targets` to the normal `cargo test` command).